connector/src/main/java/org/geysermc/connector/utils/CooldownUtils.java

```diff
@@ -149,7 +158,7 @@ public static CooldownType getByName(String name) {
                     return type;
                 }
             }
-            return DISABLED;
+            return TITLE;
```

Comment:
We shouldn't do this because legacy configs that have their cooldown settings set to false will break.